bundles/http/Citrix/AdobeSign/file.sapp

```diff
@@ -8565,7 +8568,7 @@
                 "style" : null,
                 "to" : null,
                 "serviceId" : null,
-                "value" : "https://secure.in1.echosign.com/public/agreements/view/CBJCHBCAABAAmcE3fwqIBUlMjIG4ajmOl8r8-ptq1jJt?type=esign",
+                "value" : "https://.in1.adobesign.com/public/agreements/view/CBJCHBCAABAAyFHf-sLSVHegefO1CdlH9gktcgd40O--?type=esign",
```

Comment:
Isn't the {yourAbodeSignHostname} missing? And should't the ID be masked too?

Reply:
changes done